traitsui/testing/tester/qt4/common_ui_targets.py

```diff
@@ -44,22 +44,44 @@ def register(cls, registry):
         registry : TargetRegistry
             The registry being registered to.
         """
-        handlers = [
-            (command.KeySequence,
-                (lambda wrapper, interaction: helpers.key_sequence_textbox(
-                    wrapper.target.textbox, interaction, wrapper.delay))),
-            (command.KeyClick,
-                (lambda wrapper, interaction: helpers.key_click_qwidget(
-                    wrapper.target.textbox, interaction, wrapper.delay))),
-            (command.MouseClick,
-                (lambda wrapper, _: helpers.mouse_click_qwidget(
-                    wrapper.target.textbox, wrapper.delay))),
-            (query.DisplayedText,
-                lambda wrapper, _: wrapper.target.textbox.displayText()),
-        ]
-        for interaction_class, handler in handlers:
-            registry.register_handler(
-                target_class=cls,
-                interaction_class=interaction_class,
-                handler=handler,
-            )
+        register_editable_textbox_handlers(
+            registry=registry,
+            target_class=cls,
+            widget_getter=lambda wrapper: wrapper.target.textbox,
+        )
+
+
+def register_editable_textbox_handlers(registry, target_class, widget_getter):
```

Comment:
I very much like this refactor, pulling this logic into its own function is much cleaner.  However, do you think we should move this into the helpers module, or somewhere else?  It seems a little out of place in common_ui_targets IMO, what do you think?

Reply:
I agree this function does not sit well with this module. 

But I am a bit hesitant about moving it to `helpers`. The only UITester/UIWrapper piece of knowledge in that module is the use of `interaction` object, and in the places where `interaction` is referred to, they could be replaced with more primitive data such as a string. Perhaps `helpers` should be renamed to something more meaningful to prevent it from becoming a catch-all-I-don't-know-where-this-goes kind of place.

What about... we move this `register_editable_textbox_handlers` to a new module called `registry_helper`? This will mirror the one on `tester` package and is responsible for holding functions that does some common handler registration.

Reply:
That sounds good to me!

Reply:
Also, I agree that `helpers` might be a bit too generic (all code we write should be helping with something!).  Currently all the code in `helpers` is specific to some form of interaction (keyclicks, mouse clicks, key sequences, and DisplayedText queries) so maybe just something like  `interaction_helpers` or something would be sufficient.  We can leave that for another PR

Reply:
Cool. I will move this function to a new module.
Agreed the renaming of `helpers` can be a separate PR.